hack/move-upstream.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# See HACKING.md for usage
```

Comment:
It's actually not clear what this does from reading hacking.md.  Does this help to upstream patches to kubernetes that we're carrying in origin?